### PR TITLE
LPS-156564 [Raylife D2C] Componentization and creation of active claims column chart

### PR DESCRIPTION
**Pull request by eliasantosf**
Subtask: [LPS-156564](https://issues.liferay.com/browse/LPS-156564)

### CODE REVIEW

**Comment by eliasantosf** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/dashboard-active-claims/index.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.active-claims-container .active-claims-title::after {
+	background-color: var(--primary);
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/dashboard-active-claims/index.css:2]: I believe that we have this color on Dialect.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/bar-chart/index.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
+.bar-chart{
+  flex-wrap: wrap;
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/bar-chart/index.css:2]:
```suggestion
  <div className="align-items-center bar-chart d-flex flex-wrap justify-content-between">
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/bar-chart/index.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+.bar-chart .bar-chart-title{
+  text-transform: uppercase;
+}
+
+.bar-chart .bb-chart-text{
+	color: #272833;
+	font-size: 30px;
+	font-weight: bold;
+}
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/bar-chart/index.css:13]: I believe that we have Dialect properties for this.

**Review by joanagrego — CHANGES_REQUESTED** [review]
Just a few suggestions here. Nice job!

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/bar-chart/index.css (hunk 2 above)
Comment (eliasantosf) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/bar-chart/index.css:2]: Bootstrap flex-wrap doesn't work for this class.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/bar-chart/index.css (hunk 3 above)
Comment (eliasantosf) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/bar-chart/index.css:13]: The class in question is part of the Billboard.js library, so the CSS had to be written for this class, not being possible to use Dialect in this case.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/dashboard-active-claims/index.css (hunk 1 above)
Comment (eliasantosf) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/dashboard-active-claims/index.css:2]: This color refers to the component created by after, so it is not possible to specify a color with the dialect for it in the className.

**Review by silasbsi — CHANGES_REQUESTED** [review]
Needs some adjustments. @eliasantosf

**Review by silasbsi — APPROVED** [review]

**Review by joanagrego — APPROVED** [review]
Nice!

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**